src/manifold/src/impl.cpp

```diff
@@ -902,4 +903,47 @@ SparseIndices Manifold::Impl::VertexCollisionsZ(
   else
     return collider_.Collisions<false, false>(vertsIn);
 }
+/*
+ * Returns the minimum gap between two manifolds. Returns a float between
+ * 0 and searchLength.
+ */
+float Manifold::Impl::MinGap(const Manifold::Impl& other,
+                             float searchLength) const {
+  ZoneScoped;
+  Vec<Box> faceBoxOther;
+  Vec<uint32_t> faceMortonOther;
+
+  other.GetFaceBoxMorton(faceBoxOther, faceMortonOther);
+
+  transform(autoPolicy(faceBoxOther.size()), faceBoxOther.begin(),
+            faceBoxOther.end(), faceBoxOther.begin(),
+            [searchLength](const Box& box) {
+              return Box(box.min - glm::vec3(searchLength),
+                         box.max + glm::vec3(searchLength));
+            });
+
+  SparseIndices collisions = collider_.Collisions(faceBoxOther.cview());
+
+  float minDistanceSquared = searchLength * searchLength;
+
+  for (int i = 0; i < collisions.size(); ++i) {
```

Comment:
maybe we can do parallelization here?

Reply:
I'll have a look!

Reply:
Great suggestion, it does speed it up a lot! The high res sphere [test case](https://github.com/elalish/manifold/pull/765/files#diff-9befa6164adaa593fe572551c8e23da7a628a42e0053623f1bd22f725dfabc50R902) goes from `11791 ms` to `2690 ms`!